src/pollyCLI/pollyintegration.cpp

```diff
@@ -121,6 +121,18 @@ QStringList PollyIntegration::get_projectFiles_download_url_commands(QByteArray
     return patchId;
 }
 
+bool PollyIntegration::activeInternet()
+{
+    QString command = QString("check_internet_connection");
+    QList<QByteArray> results = runQtProcess(command, QStringList()).at(0).split('\n');
+    QString status = results[1];
+    if (status == "Connected") {
```

Comment:
I am not sure if this would work. 
The `check_internet_connection` function that has been defined  in **index.js** outputs both the messages on **stdout**.  However the `status` variable that takes it value from `results[1]` is picking the value from **stderr**.  

`check_internet_connection` function should be using `console.error` when the user is not connected to internet

I have not tested this. Please correct me If I am wrong

Reply:
@rish9511 The status variable is taking the value for stdout. Stdout outputs "undefined\nConnected\n". The results[1] is the string "Connected" so it does work.

I agree with the you that console.error should be used when there is no internet connection, though. Would be cleaner.

Reply:
Oh right!
I did not read the statement 
`QList<QByteArray> results = runQtProcess(command, QStringList()).at(0).split('\n')`  carefully.

> console.error should be used when there is no internet connection, though. Would be cleaner.

Yes. And then we would not have to split the string as we are doing it now

Reply:
@rish9511 On second thoughts, console.log only has information we define. Error console could have a number of errors. Log sounds more reliable to me. What do you think?

Reply:
@shubhra-agrawal Alright, lets keep using console.log then